examples/train_resnet_base.py

```diff
@@ -0,0 +1,72 @@
+from torch_xla import runtime as xr
```

Comment:
Can you move this file to `torch_xla/examples` so it's included in our package? Then we can run examples like `python -m torch_xla.examples.train_resnet_ddp` directly from the installed package.

Reply:
do we want to include them as part of our package through?

Reply:
sure I can move.

Reply:
oh wait... we depends on torchvision in this example... I don't want to force that dependency in our package.. Let's pause this move until we figure out how to deal with that..